src/components/inputs/localized-money-input/localized-money-input.spec.js

```diff
@@ -64,23 +64,21 @@ it('should forward data-attributes', () => {
 });
 
 it('should have an HTML name', () => {
-  const { container } = renderLocalizedMoneyInput({
+  const { getByLabelText } = renderLocalizedMoneyInput({
     name: 'foo',
     selectedCurrency: 'CAD',
   });
-  expect(
-    container.querySelector('input[name="foo.CAD.amount"]')
-  ).toBeInTheDocument();
+  expect(getByLabelText('CAD')).toBeInTheDocument();
```

Comment:
this doesn't really test that it has an HTML name though?

Reply:
not directly, or indirectly :D
updated the test name